### ISSUE TITLE
Settings.ini key= gets ocaisionally overwritten. 

### ISSUE THREAD

**Issue by picanl** [tmunzer/Aerohive_Corporate_Account_Sync#7]
For some strange reason so now and then the following is happening. 

in the settings.ini the last key= value get overwritten... and looks like this....

smtpSecPassword=System.Security.SecureString
key=68 176 221 21 54 126 101 24 213 139 33 62 20 59 144 162

any idea?

**Comment by tmunzer** [comment]
Hi,

This variable is normal.
It's the configured SMTP password, but instead of keeping it in cleartext in the configuration file, I'm encrypting it under the smtpSecPassword variable.
When the script is run a first time, if the "smtpPassword" variable is present, it is encrypting it, storing the encrypted version under the "smtpSecPassword", and removing the "smtpPassword" variable.

Regards,
Thomas

**Comment by picanl** [comment]
Hi, 

I know the normal process of the encrypting. but somehow the variables get invalid value's 

smtpSecPassword=System.Security.SecureString is not a normal variable..

**Comment by tmunzer** [comment]
ok. If it's ok for you, I can add an option to keep the password in clear text in the config file.

**Comment by picanl** [comment]
Yes thats ok by me, if that fixes the issue.... It only happends once  a month  or so.

Met vriendelijke groet,

With Regards, Salutations distinguées, Mit freundlichem Gruß


Nick Hoekman

Systeem & Netwerk Beheerder

Privacy Officer (AVG/GDPR)

A: Slot 31, 7608 ND Almelo
P: Postbus 453, 7600 AL Almelo
T: 0546 - 488488
F: 0546 - 488477
M: 06 - 43832832
E: N.Hoekman@canisius.nl<mailto:N.Hoekman@canisius.nl>
I:  www.Canisius.nl<http://www.canisius.nl/>

[PiusCanisiusmail]

[IMAGE]<http://nl.linkedin.com/in/nhoekman>

[2]

Van: Thomas Munzer <notifications@github.com>
Verzonden: donderdag 6 juni 2019 10:23
Aan: tmunzer/Aerohive_Corporate_Account_Sync <Aerohive_Corporate_Account_Sync@noreply.github.com>
CC: Hoekman, N (Nick) <n.hoekman@canisius.nl>; Author <author@noreply.github.com>
Onderwerp: Re: [tmunzer/Aerohive_Corporate_Account_Sync] Settings.ini key= gets ocaisionally overwritten. (#7)


ok. If it's ok for you, I can add an option to keep the password in clear text in the config file.

-
